constants/version.go

```diff
@@ -0,0 +1,3 @@
+package constants
+
+var Version string = "Piped into LDflags on build. You are probably running Railway CLI from source."
```

Comment:
Hmm, I like this but I also don't like the fact we have to do a string compare and copy it. Maybe something like
```suggestion
const VersionDefault = "Piped into LDflags on build. You are probably running Railway CLI from source."
var Version string = VersionDefault
```

Alternatively, we can make version a private variable and have a GetVersion function which returns the (version, error) and error when the version flag isn't set

Thoughts?